easybuild/framework/easyconfig/default.py

```diff
@@ -171,6 +171,8 @@
     'moduleclass': ['base', 'Module class to be used for this software', MODULES],
     'moduleforceunload': [False, 'Force unload of all modules when loading the extension', MODULES],
     'moduleloadnoconflict': [False, "Don't check for conflicts, unload other versions instead ", MODULES],
+    'module_use_depends_on' : [False, 'Use depends_on (Lmod 7.6.1+) for dependencies in generated module '
```

Comment:
@bartoldeman I would avoid `module_use`, since that has a particular meaning (not at all related to `depends_on`).

How about `module_depends_on` or `use_depends_on`?